docs/tutorials/actions.mdx

```diff
@@ -181,8 +184,8 @@ cardano-cli conway governance action create-constitution \
   --testnet \
   --governance-action-deposit 1000000000 \
   --deposit-return-stake-verification-key-file stake.vkey \
-  --anchor-url https://tinyurl.com/3wrwb2as \
-  --anchor-data-hash "52e69500a92d80f2126c836a4903dc582006709f004cf7a28ed648f732dff8d2" \
+  --anchor-url https://github.com/Ryun1/metadata/blob/main/cip108/treasury-withdrawal.jsonld \
+  --anchor-data-hash "931f1d8cdfdc82050bd2baadfe384df8bf99b00e36cb12bfb8795beab3ac7fe5" \
   --constitution-url https://tinyurl.com/mr3ferf9  \
   --constitution-hash "5d372dca1a4cc90d7d16d966c48270e33e3aa0abcb0e78f0d5ca7ff330d2245d" \
   --governance-action-tx-id "00caeb6c2db4575acc43be3e8f87b881dccb86283daf16aa2707275cbe7f3451" \
```

Comment:
These 2 have to be changed to ` --prev-governance-action-tx-id` and `--prev-governance-action-index`

Reply:
done!
thanks for the spot @Hornan7